src/main/java/seedu/address/model/task/InternshipTodo.java

```diff
@@ -185,16 +185,16 @@ public void setNote(NoteContent note) {
     }
 
     /**
-     * Returns true if both interested internship have the same title.
+     * Returns true if both interested internship have the same title, job and deadline.
      * This defines a weaker notion of equality between two todo applications.
      */
     public boolean isSameTodo(InternshipTodo otherInternship) {
         if (otherInternship == this) {
             return true;
         }
-
         return otherInternship != null
-                && otherInternship.getInternshipTitle().equals(getInternshipTitle());
+                && otherInternship.getInternshipTitle().equals(getInternshipTitle())
+                && otherInternship.getJobTitle().equals(getJobTitle());
```

Comment:
Yea I think it might be better to compare all attributes of todo as same internship might have multiple todos

Reply:
However, idts. If we include the application deadline, it may cause ambiguity as we will have the same job position from the same company to apply but the application can be done before different dates.  It is less likely to happen in our daily life. As a reminder, we have edit content and edit deadline for our todo task, which can handle these situations more gracefully.

Reply:
The thing is, it doesn't match your JavaDoc header.

Reply:
Ok, i will change it here.